ops/travis/travis-deploy.sh

```diff
@@ -36,6 +36,12 @@ with_python27 "$BOOTSTRAP --usage-reporting=false --command-completion=false --p
 
 PATH=$PATH:$INSTALL/$NAME/bin/
 
+# Check if we need to deploy our Endpoints config - cleanup afterwards so it's not in our web deploy
```

Comment:
Can we do this after the code deploy?

That'll solve our push-safety problem, since by that point, the server will be expecting the `endpoints_sha` param to be sent to the apistatus update endpoint

Reply:
Uhh yeah we can! Good call.